uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -363,13 +378,33 @@ define([
      * Initiates a watcher over the "Allow carry forward" field.
      * Toggles dependent fields on value change.
      */
-    function watchFieldAllowCarryForward () {
+    function watchAllowCarryForwardField () {
       $scope.$watch(function () {
         return vm.fieldsIndexed.allow_carry_forward.value;
       }, function (allowCarryForward) {
         vm.fieldsIndexed.max_number_of_days_to_carry_forward.hidden = !allowCarryForward;
         vm.fieldsIndexed.carry_forward_expiration_duration.hidden = !allowCarryForward;
       });
     }
+
+    /**
+     * Initiates a watches over "Title" field.
+     * Watches for already used leave types titles.
+     */
+    function watchTitleField () {
+      var titleField = vm.fieldsIndexed.title;
+
+      $scope.$watch(function () {
+        return titleField.value;
+      }, function (title) {
+        if (title === '') {
```

Comment:
I'd use `_.isEmpty(title)` to be extra safe
```suggestion
        if (_.isEmpty(title)) {
```